packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -14,6 +14,7 @@ import 'dart:collection';
 import 'dart:ui' as ui show PointerDataPacket;
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
```

Comment:
gesture layer can't import rendering. this is layer violation

Reply:
I may need some help understanding this and how to resolve it. Thanks!